code-studio/src/js/components/AssetRow.jsx

```diff
@@ -7,7 +7,7 @@ var AssetThumbnail = require('./AssetThumbnail');
 var AssetRow = React.createClass({
   propTypes: {
     name: React.PropTypes.string.isRequired,
-    type: React.PropTypes.oneOf(['image', 'audio', 'video']).isRequired,
+    type: React.PropTypes.oneOf(['image', 'audio', 'video', 'pdf']).isRequired,
```

Comment:
Do we want to change this depending on the type of app/level?  It doesn't make a ton of sense to upload PDFs for an App Lab level.